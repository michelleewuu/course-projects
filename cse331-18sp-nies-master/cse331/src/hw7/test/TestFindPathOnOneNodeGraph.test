# Test finding path on oneNodeGraph

# Load oneNode.tsv
LoadGraph oneNodeGraph oneNode.tsv

# Find minimum-cost path on onlyNode
FindPath oneNodeGraph onlyNode onlyNode