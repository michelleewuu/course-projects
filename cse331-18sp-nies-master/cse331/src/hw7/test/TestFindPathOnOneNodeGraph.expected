# Test finding path on oneNodeGraph

# Load oneNode.tsv
loaded graph oneNodeGraph

# Find minimum-cost path on onlyNode
path from onlyNode to onlyNode:
total cost: 0.000